core/domain/html_validation_service_test.py

```diff
@@ -973,3 +978,47 @@ def test_validate_customization_args_in_tag(self):
                     soup.find(name=tag_name))))
 
         self.assertEqual(actual_output, expected_output)
+
+    def test_add_dimensions_to_noninteractive_image_tag(self):
+        test_cases = [{
+            'html_content': (
+                '<p><oppia-noninteractive-image filepath-with-value="&amp;quot;'
+                'abc.png&amp;quot;"></oppia-noninteractive-image>Hello this'
+                ' is test case to check that dimensions are added to the oppia'
+                ' noninteractive image tags.</p>'
+            ),
+            'expected_output': (
+                u'<p><oppia-noninteractive-image filepath-with-value='
+                '"&amp;quot;abc_height_32_width_32.png&amp;'
+                'quot;"></oppia-noninteractive-image>Hello this is test case'
+                ' to check that dimensions are added to the oppia '
+                'noninteractive image tags.</p>'
+            )
+        }, {
+            'html_content': (
```

Comment:
What is the difference between this and the previous test case?

You might also want to have a test case with no image tags, and another with two image tags.

Reply:
Done